panel/widgets/input.py

```diff
@@ -349,13 +349,13 @@ def _process_property_change(self, msg):
     def _process_param_change(self, msg):
         msg = super(LiteralInput, self)._process_param_change(msg)
         if 'value' in msg:
-            value = '' if msg['value'] is None else msg['value']
+            value = msg['value']
             if isinstance(value, string_types):
                 value = repr(value)
             elif self.serializer == 'json':
                 value = json.dumps(value, sort_keys=True)
```

Comment:
Don't know if you can deactivate sort_key=True since it seems to generate a bug when a point is hover and unhover quickly:
Bug with sort_key=True
![sort_key_true](https://user-images.githubusercontent.com/18531147/97810794-bf87d180-1c76-11eb-81d3-68889560caa0.gif)

without sort_key:
![sort_key_false](https://user-images.githubusercontent.com/18531147/97810804-c7e00c80-1c76-11eb-8b0e-95c04fa42ff2.gif)